src/plugins/chart_expressions/expression_xy/public/components/tooltip/tooltip.tsx

```diff
@@ -125,15 +125,17 @@ export const Tooltip: FC<Props> = ({
   const renderEndzoneTooltip = header ? isEndzoneBucket(header?.value, xDomain) : false;
 
   return (
-    <div className="detailedTooltip">
-      {renderEndzoneTooltip && (
-        <div className="detailedTooltip__header">
-          <EndzoneTooltipHeader />
-        </div>
-      )}
-      <table>
-        <tbody>{tooltipRows}</tbody>
-      </table>
-    </div>
+    <TooltipContainer>
+      <div className="detailedTooltip">
```

Comment:
Also here the `detailedTooltip` use its own euiTooltipStyle ,so probably we don't need the container

Reply:
Fixed in 8e9dce4